boxoffice/forms/order.py

```diff
@@ -35,3 +35,7 @@ class BuyerForm(forms.Form):
     email = forms.EmailField(__("Email"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
     fullname = forms.StringField(__("Full name"), validators=[forms.validators.DataRequired()])
     phone = forms.StringField(__("Phone number"), validators=[forms.validators.Length(max=16)])
+
+
+class OrderForm(forms.Form):
+    utm_campaign = forms.StringField(__("UTM Campaign"), validators=[forms.validators.Length(max=80)])
```

Comment:
You can't fail validation as there's no user input. Do what Hasjob does instead, capturing whatever is in the request and letting the database validate if the capture had an error.